frontend/src/container/TopNav/DateTimeSelectionV2/index.tsx

```diff
@@ -0,0 +1,474 @@
+import './DateTimeSelectionV2.styles.scss';
```

Comment:
We have copied all the functionality from current date selector to this file right?

Reply:
yes